### PR TITLE
second part 2 of Cloud Talent solution v4

### PR DESCRIPTION
**Pull request by munkhuushmgl**
There is no unit test for CreateCLientEvent because it exposes full path of jobs.

### CODE REVIEW

File: jobs/v4/src/main/java/com/example/jobs/JobSearchCommuteSearch.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+import com.google.cloud.talent.v4beta1.SearchJobsRequest;
+import com.google.cloud.talent.v4beta1.SearchJobsResponse;
+import com.google.cloud.talent.v4beta1.TenantName;
+import com.google.cloud.talent.v4beta1.TenantOrProjectName;
+import com.google.protobuf.Duration;
+import com.google.type.LatLng;
+
+import java.io.IOException;
+
+public class JobSearchCommuteSearch {
```
Comment (lesv) [inline jobs/v4/src/main/java/com/example/jobs/JobSearchCommuteSearch.java]: nit - class name seems a bit wordy.  JobByCommuteSearch perhaps.  (not required)

Comment (munkhuushmgl) [reply jobs/v4/src/main/java/com/example/jobs/JobSearchCommuteSearch.java]: Only class or all other search classes in general? I can remove the prefix [JobSearch] from Search classes
